test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests/NuGetPackageManagerTelemetryTests.cs

```diff
@@ -639,6 +649,16 @@ public override void EmitTelemetryEvent(TelemetryEvent telemetryData)
                 _telemetrySession.PostEvent(telemetryData);
             }
         }
+
+        private static SourceRepository CreateMockSourceRepository(string sourcePath)
```

Comment:
can't we just use the standard source repository? or is there a benefit here I'm missing.

Reply:
Actually - these tests don't use the SourceRepository, anyway, so I'll remove that